mockito-kotlin/src/main/kotlin/com/nhaarman/mockitokotlin2/OngoingStubbing.kt

```diff
@@ -26,10 +26,12 @@
 package com.nhaarman.mockitokotlin2
 
 import org.mockito.Mockito
+import org.mockito.internal.invocation.InterceptedInvocation
 import org.mockito.invocation.InvocationOnMock
 import org.mockito.stubbing.Answer
 import org.mockito.stubbing.OngoingStubbing
-import kotlin.DeprecationLevel.ERROR
+import kotlin.coroutines.Continuation
+import kotlin.coroutines.intrinsics.startCoroutineUninterceptedOrReturn
```

Comment:
I believe this has now been moved to an `*.experimental.*` package in the latest version of Kotlin.

https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.coroutines.experimental.intrinsics/start-coroutine-unintercepted-or-return.html

Reply:
It looks like it has since been moved back again: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.coroutines.intrinsics/start-coroutine-unintercepted-or-return.html

Reply:
Yes. The same code was working for all coroutine and kotlin versions. Now I am using `1.4.0` with `1.3.9` of coroutines.

Reply:
You might consider replacing startCoroutineUninterceptedOrReturn with it's implementation if the definition location isn't very stable. All it does is cast to expose the continuation parameter and invoke.
```
(answer as ((InvocationOnMock, Continuation<T?>) -> Any)).invoke(it, continuation)
```